libcoap-sys/build/build_system/vendored.rs

```diff
@@ -237,14 +250,14 @@ impl VendoredBuildSystem {
         } else {
             // SAFETY: We are still single-threaded here.
             unsafe { env::set_var("PKG_CONFIG_PATH", pkg_config_path_bak.unwrap_or_default()) }
+            println!("cargo:rustc-link-lib=static=coap-3");
             println!(
                 "cargo:rustc-link-search={}",
                 libcoap_build_prefix
                     .join("lib")
                     .to_str()
                     .context("unable to convert OUT_DIR to a valid UTF-8 string.")?
             );
-            println!("cargo:rustc-link-lib=static=coap-3");
```

Comment:
Is it intentional that the sequence has changed here?

Reply:
Was just part of my experimenting around, shouldn't make a difference.